homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -212,8 +207,8 @@ class PropertyZWaveJSEntityDescription(BinarySensorEntityDescription):
     NotificationZWaveJSEntityDescription(
         # NotificationType 6: Access Control - State Id 22 (door/window open)
         key=NOTIFICATION_ACCESS_CONTROL,
-        off_state="23",
-        states=("22", "23"),
+        not_states={"23"},
```

Comment:
Can you clarify why you changed off_states into not_states because this may mean something else:
By specifically specifying which values are off and on, you can treat any other value as unknown.